src/game/default/g_ballistics.c

```diff
@@ -315,6 +315,69 @@ static void G_GrenadeProjectile_Explode(g_entity_t *self) {
 			G_BurnMark(self->s.origin, plane, surf, 20);
 		}
 	}
+	
+	// attempt to stop the sound playing...doesnt work
+	self->s.sound = 0;
+	G_FreeEntity(self);
+}
+
+/*
+ * @brief mostly a copy of the grenade launcher version but with different
+ * means of death messages
+ */
+static void G_HandGrenadeProjectile_Explode(g_entity_t *self) {
```

Comment:
We could probably use the same function here, and do a strncmp on self->class_name to determine the means of death.